projects/plugins/jetpack/readme.txt

```diff
@@ -284,8 +284,9 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 == Changelog ==
 ### 12.2-beta - 2023-05-30
 #### Major Enhancements
-- Add an AI Assistant block, replacing the previous experimental AI Paragraph and AI Image blocks.
+- New AI Assistant block: harness AI power directly from your editor.
 - Donations, Paid Content and Payment Buttons blocks are now available with all Jetpack plans. A commission is charged for free plans.
+- Social: automatically share your new posts to Mastodon and Instagram.
```

Comment:
Now that I look closer, let's make these changes in Jetpack's CHANGELOG.md, either instead or additionally. Otherwise this is liable to get accidentally overwritten when I update the changelog for all the other cherry picks.